pkg/asset/templates/kube-apiserver.yaml

```diff
@@ -32,7 +32,6 @@ spec:
         - --bind-address=0.0.0.0
         - --secure-port=443
         - --insecure-port=8080
-        - --advertise-address=$(MY_POD_IP)
```

Comment:
IIRC this was because with vagrant/virtualbox the default interface is not routable between vms (I would need to re-verify this). Given that the vagrant support is only for local clusters, i'd be okay resolving this in some other way to make our manifests more generic.